contracts/evoting/admin_test.go

```diff
@@ -0,0 +1,45 @@
+package evoting
+
+import (
+	"github.com/c4dt/d-voting/contracts/evoting/types"
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/dela/serde"
+	sjson "go.dedis.ch/dela/serde/json"
+	"testing"
+)
+
+var ctxAdminTest serde.Context
+
+var formFacAdminTest serde.Factory
+var transactionFacAdminTest serde.Factory
+
+func init() {
+	ciphervoteFac := types.CiphervoteFactory{}
+	formFacAdminTest = types.NewFormFactory(ciphervoteFac, fakeAuthorityFactory{})
+	transactionFacAdminTest = types.NewTransactionFactory(ciphervoteFac)
+
+	ctxAdminTest = sjson.NewContext()
+}
+
+func TestAdmin_serde(t *testing.T) {
```

Comment:
I think it's helpful to add a comment to explain what you're testing, that makes it easier to get into the context when debugging failing tests

Reply:
Indeed will add it